### DISCUSSION TITLE
Banned twice on discord for no reason

### DISCUSSION THREAD

**Discussion by KumaKuma** [younesaassila/ttv-lol-pro#171] [General]
I was banned, TWICE on the discord for reporting an issue on the plugin for no apparent reason whatsoever. I reported I was getting consistent buffering issues with the new release but not with the 2.0.0 pre releases. The only thing I can think of for being banned is uploading a file but it was literally just the 2.0.0 prerelease. 
What the hell man?

**Comment by zGato** [comment]
TTV LOL (https://ttv.lol) discord server has nothing to do with TTV LOL PRO. We're not affiliated.
They can ban you, and we can't do anything to prevent it.

**Comment by younesaassila** [comment]
This bug will be fixed in v2.0.1. As @zGato said, we don't manage the TTV LOL Discord server. I don't know why you got banned and can't unban you
